README.md

```diff
@@ -21,6 +21,153 @@ as these things will be provided by your cloud provider.
 
 But in case of a local deployment, navigate to the next section.
 
+#### Setting up remote cluster
+
+##### Setting up custom error backend
+
+On remote environment each namespace should have a custom error backend setup for the nginx. 
+This is done by creating the following ConfigMaps and Services in the namespace:
+
+This only needs to be applied once per namespace.
+
+Configuration for the default backend: service and handling of a 404 error on a subdomain level (wild card domain).
+
+Create `custom-error-backent.yaml` file with the following content. 
+Don't forget to change the host name for the wildcard in case you are deploying the environment other than production.
+
+<details>
+  <summary>Click to see the content of the custom-error-backend.yaml file</summary>
+
+    ```yaml
```

Comment:
I think the indentation is incorrect for this code fencing. Does not look right when looking at the rendered README.